src/components/DonationWidget/components/DonationAddressForm.tsx

```diff
@@ -19,6 +19,21 @@ const DonationAddressForm: React.FC<Props> = ({
   onEditAmount,
   onSubmit
 }) => {
+  const handleOnSubmit = async (e: React.ChangeEvent<HTMLFormElement>) => {
+    e.persist();
```

Comment:
Nice, what is this for? just curious

Reply:
Actually it is the first time I need it to use it. [Better explanation at this medium post](https://medium.com/trabe/react-syntheticevent-reuse-889cd52981b6) but is related to the way I am accessing/using the `event.target`. What is odd for me is that I only really need it now on the `donateOnce` and `donateMonthy` submissions, but I can't spot how are they different from others.

For now, I add it on all the forms to keep it as a safety need to avoid any further issues.